CounterApp/script.js

```diff
@@ -0,0 +1,14 @@
+
+        const buttonMinus= document.getElementById("minus-button")
+        const buttonPlus= document.getElementById("plus-button")
+        const span= document.getElementById("value")
+        
+        let value= 0;
+        buttonMinus.addEventListener("click", ()=>{
+            value--;
+            span.textContent= value;
```

Comment:
o `=` ta colado na propriedade
```suggestion
            span.textContent = value;
```